tests/e2e/e2e_test.go

```diff
@@ -193,14 +195,14 @@ var _ = Describe("Usecase #5", func() {
 		It("should push reflect changes", func() {
 
 			// Get IP and port
-			getIP := runCmd("oc get svc nodejs -o go-template='{{.spec.clusterIP}}:{{(index .spec.ports 0).port}}'")
-			pingUrl := fmt.Sprintf("http://%s", getIP)
+			// getIP := runCmd("oc get svc usecase5-nodejs -o go-template='{{.spec.clusterIP}}:{{(index .spec.ports 0).port}}'")
+			// pingUrl := fmt.Sprintf("http://%s", getIP)
 
-			pingSvc(pingUrl)
-			grepBeforePush := runCmd("curl -s " + pingUrl +
-				" | grep 'Welcome to your Node.js application on OpenShift'")
+			// pingSvc(pingUrl)
+			// grepBeforePush := runCmd("curl -s " + pingUrl +
+			// " | grep 'Welcome to your Node.js application on OpenShift'")
 
-			log.Printf("Text before change: %s", strings.TrimSpace(grepBeforePush))
+			// log.Printf("Text before change: %s", strings.TrimSpace(grepBeforePush))
```

Comment:
By commenting out this section, we won't be doing a `curl` before updating the html file. So there won't be any before and after verification. Are we okay with that?

Reply:
@ashetty1 There still is, we run sed before pushing: `https://github.com/redhat-developer/ocdev/pull/303/files/f626589e6eff5cc19b89453adc6ab26581247275#diff-73bf4cf8a09657b595c016b77c1d805bR208`

Reply:
the problem is that pinging doesn't work using `minishift` since the IP address is private and inaccessible.

Reply:
@cdrage ah! That's a good enough reason I guess :+1:

Might make sense to add a note there? I will see if we can do it in any other way.